gateway/conf/samples/root.yml

```diff
@@ -1,35 +1,35 @@
 $namespace avalanche:
   configurationPath: avalanche.yml
-  schemaPath: ethereum-schema.json
+  schemaPath: ../../src/services/schema/ethereum-schema.json
```

Comment:
The relative path should start from the schema directory, so there should be no need to change any of these.

When you're given the flexibility to design something for users, you should strive to make things simple. If you came up with a design that requires adding the same path prefix to entry - it's a good sign that the path prefix should not be there.

Also, make sure to test your updated config samples to make sure it works.